src/plugins/intel_cpu/src/nodes/executors/dnnl/dnnl_matmul_primitive.cpp

```diff
@@ -132,7 +153,30 @@ static DnnlPrimitiveAttrs createPrimitiveAttrs(const MatMulAttrs& attrs,
     DnnlPostOpsComposer
         dnnlpoc(postOps, context->getEngine(), dims, dims.size() - 1, isINT8, 1 << 0, memory, outputDataType);
 
-    return dnnlpoc.compose();
+    const auto maxRank =
+        std::max({srcDesc->getShape().getRank(), weiDesc->getShape().getRank(), dstDesc->getShape().getRank()});
+    auto normWeiDims = normalizeToRank(weiDesc->getShape().getStaticDims(), maxRank);
```

Comment:
Minor
```suggestion
    const auto normWeiDims = normalizeToRank(weiDesc->getShape().getStaticDims(), maxRank);
```

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/27459/commits/51b67cac8980e4c8f9cf60a357547563e774151a